app-office/moneydance/Manifest

```diff
@@ -1,3 +1,4 @@
 DIST moneydance-2017.10-amd64.tar.gz 132116528 BLAKE2B f5ba44a89944657c6c1380226dc512729825b084783e6d70dd7b2b731411294c7333bae858b2be3d5321e5145230257b325dc2b72a22744a877673c1c934d854 SHA512 175ff470b5fb5c3851eae708cc641777d7bdf1ede40366ccf5a135d6c813c08d9ce32665a2307c3aefa63eeb2206ad0874cb989aa998415abb86184a2b114ce8
 DIST moneydance-2017.10-x86.tar.gz 134860964 BLAKE2B 92181a6c6a81063377823b0fd9ed652036cf1f633c2cab57f19d1dbb18a2542361fc2bafab8852c11b941a24532f662351ea2d05bc928818db24a4cf3cae7112 SHA512 18d9d4ce7b0b65566ac81fea66ba34d66173559ca90ac166cc9b90631400845ae1c9d92ae781a618cacc57cd4f630f079736101d29b66d9c6cee7d32c7c15c22
 DIST moneydance-2019.1-amd64.tar.gz 132469388 BLAKE2B 516ff3af46e23370cf97b7a41e9802f14f5272c36b01080d69f339bdb9ca7de045de35baead8d97a8febece062dcd8b2befecad2744773e9a1c79b9a16b9639a SHA512 a28787dcc10173f4d492ec1bd5a8f17b6592f86c324e57a7e2951a0265825e4f78b620a4aabb4cde94c97860d3aaac8066572882c772d5abfa79c90cd33770ee
+DIST moneydance-2019.2-amd64.tar.gz 134406118 BLAKE2B 248ae5c43e5129348dfbfd518563eb4af8d296abbf42f6290453914a07c5f110d08a18f83182233d0612d5ea3eee613cba75436fd273a80af66471bb2e4a5ef4 SHA512 66a4b89b3826eda4079554c1dd916cc81082eb6a5dd2876a12986ae92d84a5875dda1df44c349dc5ae90a2ec9059452325c3ebde4001264de81fc5c81794668d
```

Comment:
```
>>> Emerging (1 of 1) app-office/moneydance-2019.2::gentoo
Refetching... File renamed to '/tmp/dist/moneydance-2019.2-amd64.tar.gz._checksum_failure_.9_fyf0f4'
```

So I suppose that's no longer the newest version.

Reply:
Yes. I fixed this now, since I found out, how to get the specific version links. Also bumped directly to 2019.3.

What about 2017? Should I motify it, to match also those specific version urls?

Reply:
Sure, if they have matching hashes.